src/tmplits/tmplits-tablerow/module.js

```diff
@@ -27,7 +27,7 @@ export function TmplitTableRow(context, args) {
         attr
     } = util.cleanArgs(_args)
 
-    classList = classList.concat(['tmplit-table-row'])
+    classList.push('tmplit-table-row');
```

Comment:
```suggestion
    classList.push('tmplit-table-row')
```
Same style nitpick here about semi-colon consistency. It's functionally the same.